### ISSUE TITLE
The TeX source of the font tables

### ISSUE THREAD

**Issue by jsbien** [stipub/stixfonts#247]
What about making the source of the font tables publicly available? I would be happy to see it and to mention it in my paper submitted to TUGboat (entitled "Creating annotated Unicode-like font charts").

**Comment by tiroj** [comment]
The fonts are not developed with TeX, so I am unsure exactly what you mean. Can you elaborate?

The full sources of the STIX Two fonts that we use in development are available. The sources for individual tables are various because of the complexity of the workflow and toolchain, so one needs to read the build documentation carefully to see where to find the input source for the tables. Basically:

a) The glyph outlines and sources for other common OT format tables are in the FontLab sources, but for build purposes these are written out to UFO format, which is also a common interchange format for most font development tools.

b) The source for the OpenType Layout tables (GSUB, GPOS, GDEF) are the .volt.ttf files, which are TTFs with custom tables storing the MS VOLT (Visual OpenType Layout Tool) project data; for the build process these are compiled and ‘shipped’ to the input.ttf files from where these tables are grabbed when the build tool is run.

c) The format 14 cmap subtable has to be manually edited (e.g. with a TTX .xml dump or a table editor tool like OTMaster), and resides in the .input.ttf file for the Math font.

d) The MATH table is its own source: the MS MathEditor tool works directly on the table rather than via a higher-level source format.

The toolchain we use to develop the STIX Two fonts involves some proprietary software such as FontLab and VOLT, but we try to be flexible in terms of ensuring that the sources could be converted to use by other, open source tools. So the build tool itself is open source as are all the tools and libraries that it utilises; and we make some [open source font data management and conversion tools](url) available.

**Comment by jsbien** [comment]
Thank you very much for your detailed answer.

For STIXTwoMath-Regular.pdf and STIXTwoText-Regular.pdf evince shows "Creator: LaTeX with hyperref package" and "Producer: XeTeX 0.99996". I'm interested in the XeLaTeX source which was used to produce these documents.

**Comment by davidmjones** [comment]
I think he means the STIXTwoMath-Regular.pdf and STIXTwoText-Regular.pdf charts in the docs directory.  In principle I would like to make the sources available, but the LaTeX files are generated by a perl script that is somewhat hand-crafted for the STIX fonts, so I'm not sure how useful any of it would be.  I'm also in no position to provide any support of any sort for it right now.

@jsbien if you want to contact me directly, I'd be happy to tell you more about how the charts are generated and we can see if there's anything useful for you.

**Comment by jsbien** [comment]
@davidmjones you are right and I would be glad to contact you directly. As you can see at https://github.com/jsbien, my mail is jsbien@mimuw.edu.pl.
